orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/render/YamlRenderedValueConverter.java

```diff
@@ -67,4 +70,12 @@ private static boolean containsEL(String renderedValue) {
   private static boolean isYamlKeyword(String renderedValue) {
     return YAML_KEYWORDS.contains(renderedValue.toLowerCase());
   }
+
+  private static boolean containsNoConvertMarker(String renderedValue) {
+    return renderedValue.startsWith("noconvert:");
```

Comment:
Do you want the marker to be `noconvert` or `noexpand` ?  You have `noconvert` here but your PR comment uses `noexpand`.

Reply:
Whoops. Which do you prefer?

Reply:
`noexpand`